docs/cloud/rahti/usage/projects_and_quota.md

```diff
@@ -53,73 +62,32 @@ click on "Save". After doing this, the default billing project
 instructions for setting up oc in the [command line tool usage
 instructions ](/cloud/rahti/usage/cli). For example, if your OpenShift project is called
 *my-openshift-project*, you would run:
 
 ```bash
-oc get project my-openshift-project -o yaml
+oc get project my-openshift-project -o yaml | grep csc_project
 ```
 
 This should produce the following output:
 
 ```yaml
-apiVersion: project.openshift.io/v1
-kind: Project
-metadata:
-  annotations:
-    ...
-  creationTimestamp: 2018-11-22T12:27:05Z
-  labels:
     csc_project: "1000123"
-  name: my-openshift-project
-  resourceVersion: "72557736"
-  selfLink: /apis/project.openshift.io/v1/projects/my-openshift-project
-  uid: df4970e2-abd7-4417-adbf-531293c68cd6
-spec:
-  finalizers:
-  - openshift.io/origin
-  - kubernetes
-status:
-  phase: Active
 ```
 
-In the output above, you can find the associated CSC computing project under
-`metadata.labels.csc_project`. In this case, the project is `1000123`.
-Unfortunately, this information is not available via a web interface yet.
+In this case, the project is `1000123`. Unfortunately, this information is not available via a web interface.
 
 !!! note
     It is not possible for normal users to change the *csc_project* label
     after a project has been created. If you would like to change the label for
     an existing project, please [contact the support](/support/contact/). You can also create
     a completely new project if you want to use a different label.
 
-## Creating a project
-
-Click the blue "Create Project" button to create a project, and you will be
-presented with the following view:
-
-![OpenShift new project dialog](img/new_project_dialog_3.7.png)
-
-1. You *need* to pick a **unique name** that is not in use by any other project
-in the system.
-1. You *can* also enter a **human-readable display name** and.
-1. You *have to* also enter a **CSC computing project** in the _Description_ field. It must be a currently valid CSC project, that your account has access to. In order to view to which CSC projects you have access to, please check <https://my.csc.fi>. If you have access to no CSC project, you will not be able to create any Rahti project. If you have Rahti access via project_1000123, you would enter the following in the Description field:
-
-> csc_project: XXXXXXX
-
-See the section about [accounts](/accounts/).
-
-Once you have filled in the fields, click "Create", and you will see the application
-catalog where you can pick an application template or import your
-own one.
-
-For more information about using the web interface, refer to the
-[official OpenShift documentation](https://docs.okd.io/). You can find
-out which version of the documentation to look at in the web interface by
-clicking the question mark symbol in the top bar and selecting "About".
+!!! note
+    It is possible to change the description of a project, but **it will not** change the csc_project label**.
```

Comment:
There is some formatting error here. Which part should in in **bold text** ?

Reply:
Second bold was unnecessary. I added the word billing explicitly.\

thank you for the review